File: series1_handout/series1_handout/trapezoidal/exercise1e_plot.m
u1 = load('u1.txt');
u2 = load('u2.txt');
u3 = load('u3.txt');
u4 = load('u4.txt');
t = load('time.txt');

plot(t, u1, t, u2, t, u3, t, u4)
legend('u_1', 'u_2', 'u_3', 'u_4')
figure()
plot(t, u1+u2+u3+u4)
legend('u_1+u_2+u_3+u_4')

File: series1_handout/series1_handout/trapezoidal/exercise1e_plot.py
import numpy
import pylab


u1 = numpy.loadtxt("u1.txt")
u2 = numpy.loadtxt("u2.txt")
u3 = numpy.loadtxt("u3.txt")
u4 = numpy.loadtxt("u4.txt")
t = numpy.loadtxt("time.txt")

pylab.plot(t, u1, label="u1")
pylab.plot(t, u2, label="u2")
pylab.plot(t, u3, label="u3")
pylab.plot(t, u4, label="u4")
pylab.legend()
pylab.show()

pylab.plot(t, u1+u2+u3+u4, label="u1+u2+u3+u4")
pylab.ylim([0,1.5])
pylab.legend()
pylab.show()
